src/components/Settings.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <button type="button" class="nes-btn is-primary" @click="exportData()">
+    Export data
+  </button>
+
+  <button type="button" class="nes-btn is-primary" @click="importData()">
+    Import data
+  </button><br/><br/>
+
+  <textarea class="nes-textarea" rows="10" v-model="topics"></textarea>
+</template>
+
+<script>
+import { Vue } from 'vue-class-component';
+import { store } from '@/state';
+
+export default class Settings extends Vue {
+  state = store.state;
+  topics = '';
+
+  get currentTopics () {
```

Comment:
if those are only needed to iterate over - these could be just used in the method with `this.state.topics`